src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/StatusAlert.jsx

```diff
@@ -11,9 +15,11 @@ export default function StatusAlert({ appointment, facility }) {
 
   const canceled = appointment.status === APPOINTMENT_STATUS.cancelled;
   const isPastAppointment = appointment.vaos.isPastAppointment;
-  const canceler = appointment.description?.includes('CANCELLED BY PATIENT')
-    ? 'You'
-    : facility?.name || 'Facility';
+  const canceler =
+    appointment.description?.includes('CANCELLED BY PATIENT') ||
```

Comment:
My hope with this ticket was that we could update both v1 and v2 transformers to set `cancellationReason`, and not have to check the description here.

Reply:
So when you say, 'set the cancellationReason', you mean the actual string text "You cancelled this appointment" or "Facility cancelled this appointment"? If so, getting the facility information in the transformer seems to be a problem.

Reply:
I mean that the appointments that are returned from both the v1 and v2 transformers have a `cancellationReason` field set to `pat` or `prov` and this component checks only `cancellationReason`.